How Mattermost stores Active Directory ObjectGUID. (#3733)
mattermost/docs: source/install/troubleshooting.rst

- ``too soon`` or ``too late``: Assertion ``NotOnOrAfter`` or ``NotBefore`` attribute outside current time.

``Why does the objectGUID of a user in Mattermost differ from what we're seeing in ADFS``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> The Active Directory Object-Guid attribute LDAP display name ``objectGUID`` is a 16 byte array which can be displayed in different ways:

Reviewer: ```suggestion
The Active Directory ObjectGUID attribute LDAP display name ``objectGUID`` is a 16-byte array which can be displayed in different ways:
```

> - The ``ldapsearch`` linux command displays it as base 64: ``Hrz/HqNKnU+lCNTYHx9Ycw==``. This is also the format used in LDIF files.

Reviewer: ```suggestion
- The ``ldapsearch`` Linux command displays it as base64: ``Hrz/HqNKnU+lCNTYHx9Ycw==``. This is also the format used in LDIF files.
```

> - The `LDAP Golang package we use <https://github.com/go-ldap/ldap>`__ shows the value as hexidecimal (base 16) array with each byte separated by a backslash: ``\1e\bc\ff\1e\a3\4a\9d\4f\a5\08\d4\d8\1f\1f\58\73``

Reviewer: ```suggestion
- The `LDAP Golang package we use <https://github.com/go-ldap/ldap>`__ shows the value as hexadecimal (base 16) array with each byte separated by a backslash: ``\1e\bc\ff\1e\a3\4a\9d\4f\a5\08\d4\d8\1f\1f\58\73``.
```

> You can remove the backslashes: ``1ebcff1ea34a9d4fa508d4d81f1f5873`` and parse it with `Golang <https://play.golang.org/p/9b8iDPuz0Nm>`__ as snippets which prints the base 10 representation of each value: ``30 188 255 30 163 74 157 79 165 8 212 216 31 31 88 115``

Reviewer: ```suggestion
You can remove the backslashes: ``1ebcff1ea34a9d4fa508d4d81f1f5873`` and parse it with `Golang <https://play.golang.org/p/9b8iDPuz0Nm>`__ as snippets which prints the base 10 representation of each value: ``30 188 255 30 163 74 157 79 165 8 212 216 31 31 88 115``.
```

> - Windows powershell displays the value like this: ``1effbc1e-4aa3-4f9d-a508-d4d81f1f5873``

Reviewer: ```suggestion
- Windows PowerShell displays the value like this: ``1effbc1e-4aa3-4f9d-a508-d4d81f1f5873``.
```
